packages/deployment/app/cloudformation.template.yml

```diff
@@ -1067,26 +1067,26 @@ Resources:
       Metrics:
         - Id: e1
           Expression: "FILL(m1, 0)"
-          Label: NormalizedActivityScheduleTime
+          Label: Normalized{{this.scaleIn.metricName}}
```

Comment:
This custom metric that used the `FILL()` function was required when we were using the ActivityScheduleTime for scaling. It's likely that, for most other metrics, users will not want to fill missing data points with "0".

Reply:
I think it's reasonable to fill the missing data with 0 for autoscaling purpose.  Even for custom metrics, we still need the data point to trigger the alarm.